### PR TITLE
Docs for Tinymath Functions

### PR DESCRIPTION
**Pull request by cqliu1**
Adding doc blocks for each math function in Tinymath. RE: issue #18.

### CODE REVIEW

**Comment by cqliu1** [comment]
Close #18

File: src/functions/exp.js [hunk 1]
```diff
@@ -1,3 +1,13 @@
+/**
+ * Calculates the exponent of a number. For arrays, the function will be applied index-wise to each element.
```
Comment (rashidkpc) [inline src/functions/exp.js]: This seems like a better description: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/exp

File: src/functions/fix.js [hunk 2]
```diff
@@ -5,6 +5,17 @@ const fixer = a => {
...
 };
 
+/**
+ * Calculates the fix of a number, i.e. rounds a number towards 0. For arrays, the function will be applied index-wise to each element.
+ * @param {(number|number[])} a a number or an array of numbers
+ * @return {(number|number[])} The fix of `a`. Returns an array with the the fixes for each element if `a` is an array.
+ *
+ * @example
+ * fix(1.2) // returns 1
+ * fix(-1.8) // returns 1
```
Comment (rashidkpc) [inline src/functions/fix.js]: Typo

File: src/functions/floor.js [hunk 3]
```diff
@@ -1,3 +1,13 @@
+/**
+ * Calculates the floor of a number, i.e. rounds a number towards negative infinity. For arrays, the function will be applied index-wise to each element.
+ * @param {(number|number[])} a a number or an array of numbers
+ * @return {(number|number[])} The floor of `a`. Returns an array with the the floor of each element if `a` is an array.
+ *
+ * @example
+ * floor(1.2) // returns 1
```
Comment (rashidkpc) [inline src/functions/floor.js]: Add a negative example

File: src/functions/ceil.js [hunk 4]
```diff
@@ -1,3 +1,13 @@
+/**
+ * Calculates the ceiling of a number, i.e. rounds a number towards positive infinity. For arrays, the function will be applied index-wise to each element.
+ * @param {(number|number[])} a a number or an array of numbers
+ * @return {(number|number[])} The ceiling of `a`. Returns an array with the the ceilings of each element if `a` is an array.
+ *
+ * @example
+ * ceil(1.2) // returns 2
```
Comment (rashidkpc) [inline src/functions/ceil.js]: Add a negative example

File: src/functions/count.js [hunk 5]
```diff
@@ -1,4 +1,17 @@
...
+/**
+ * Returns the length of an array. Alias for size
+ * @param {any[]} a array of any values
+ * @return {(number)} The length of the array. Returns 1 if `a` is not an array.
+ * @throws `'Must pass an array'` if `a` is not an array
+ * @example
+ * count([]) // returns 0
+ * count([-1, -2, -3, -4]) // returns 4
+ * count(100) // returns 1
+ * count('foo') // returns 1
```
Comment (rashidkpc) [inline src/functions/count.js]: I get why this does this, but can we remove this example. We largely don't handle strings.

File: src/functions/last.js [hunk 6]
```diff
@@ -1,3 +1,14 @@
+/**
+ * Returns the last element of an array. If anything other than an array is passed in, the input is returned.
+ * @param {any[]} a array of any values
+ * @return {*} The last element of 'a'. Returns `a` if `a` is not an array.
+ *
+ * @example
+ * last(2) // returns 2
+ * last('foo') // returns 'foo'
```
Comment (rashidkpc) [inline src/functions/last.js]: Same deal here, I don't want to document the results of what amounts to a side effect.

**Comment by rashidkpc** [comment]
Left a few comments, but otherwise LGTM

**Pull request merged**